packages/date/src/DatepickerInput.spec.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-global-assign -- need for test purposes */
+import React from 'react';
+
+import { render, configure } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import timezonedDate from 'timezoned-date';
+
+import { DatepickerInput } from './DatepickerInput';
+import { userInputFromDatetime } from './utils/date';
+
+configure({
+  testIdAttribute: 'data-test-id',
+});
+
+let originalDate: typeof Date;
+beforeEach(() => {
+  originalDate = Date;
+});
+afterEach(() => {
+  Date = originalDate;
+});
+
+const renderDatepicker = (dateString: string) => {
+  const { date } = userInputFromDatetime({
+    value: dateString,
+    uses12hClock: false,
+  });
+
+  return render(<DatepickerInput value={date} onChange={jest.fn()} />);
+};
+
+describe('Date: DatepickerInput', function () {
+  describe('renders correct date, when the system UTC offset is +02:00 and the one in the props.value is', () => {
+    it('+04:00', async () => {
+      Date = timezonedDate.makeConstructor(120);
+      const { getByTestId } = renderDatepicker('2022-09-22T00:00+04:00');
```

Comment:
Just a suggestion, but it's worth reconsidering this pattern. It would fail this eslint rule: https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-screen-queries.md